client/blocks/image-editor/image-editor-toolbar.jsx

```diff
@@ -179,21 +208,24 @@ class ImageEditorToolbar extends Component {
 			}
 		];
 
-		return buttons.map( button =>
-			button
+		return buttons.map( button => {
```

Comment:
Okay. Good to know for future reference. 

So`Array.prototype.some|every|filter` and other IE9+supporting native methods can be safely used, I take it? 

Just the `include(), `keys()` and others should be lodash(ed)?